### ISSUE TITLE
Maintain jQuery's matched element state

### ISSUE THREAD

**Issue by colllin** [premasagar/important#5]
First of all, thank you for developing this plugin!  I'm using it in production for a [crossfit workout tracker](http://cfwhiteboard.com) and it's working great.

I ran into an issue today where I was trying to do something like:

``` javascript
$('li:first')
    .add('li:last').css({
        color: 'red'
    })
    .end()[0].clientWidth
```

where the .clientWidth part would throw a `TypeError: Cannot read property 'clientWidth' of undefined`.  So the push state of matched elements is not being maintained for chaining purposes.  Is this difficult to fix?  In the meantime I'm using jQuery.important.noConflict() and apply()ing the `css` function where I need it.

**Comment by premasagar** [comment]
Hmm, good catch, @collindo. The `end()` method should return the previous collection, as it would do without the plugin.

You could avoid this problem by avoiding use of `end()`, e.g. 

``` js
var first = $('li:first');
first.add('li:last').css({
        color: 'red'
    });
first[0].clientWidth
```

but it'd be good if you didn't have to think about it.

I'm not actively using the library at the moment, so my development energy isn't currently going into it. I wouldn't like to say if I'll get around to implementing a fix for this, but I'm happy to take pull requests.

**Comment by colllin** [comment]
Good point.  Seems like it could be avoided pretty easily.  I'm actively
using it, but I already worked around it on my end.  I'll send you a pull
request when I get a chance.

**Comment by premasagar** [comment]
Cool!
